src/redis.cr

```diff
@@ -43,6 +43,7 @@ class Redis
   # :nodoc:
   alias Request = Array(RedisValue)
 
+  getter! location : String
```

Comment:
They have `url`, `location` and `id` methods.  I'm not clear what each means but `url` is fine with me.  It is a legal URL so that works.